automatic/wps-office-free/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,16 @@
+  $packageName = 'wps-office-free'
+  $regName = $packageName -replace('\-',' ')
+  $regName = $regName -replace('free','*')
+  $registry = Get-UninstallRegistryKey -SoftwareName $regName
+  $file = $registry.UninstallString
+  
+  # All arguments for the Uninstallation of this package
+  $packageArgs = @{
+  PackageName = $packageName
```

Comment:
Indentation of block 2 spaces as per [§2.2](https://github.com/chocolatey/chocolatey-coreteampackages/wiki/Contributing-guidelines#22-formatting)